2025.html

```diff
@@ -73,6 +73,7 @@ <h3>Kudos to these websites who <!-- TODO: got --> plan to get naked in 2025!</h
             <li><a href="https://davidroessli.com/">David Roessli</a></li>
             <li><a href="https://omz13.com/">David Somers</a></li>
             <li><a href="https://silvanzwick.com/">Silvan Zwick</a></li>
+            <li><a href="https://tommi.space/" title="The virtual representation of Tommi’s mind 🤯">tommi.space</a></li>
```

Comment:
1. Please remove the `title` (appreciated, but we don’t use it for any other link)
2. Optional: Please consider using your name as link text
3. Please sort alphabetically (even if you keep “tommi.space,” go by the lastnames in the list, so right now this should go after David Somers)

Reply:
I did 1. and 3. but I prefer not to share my full name there.